### PR TITLE
Update text for 'force:apex:test:run' command in VS Code

### PR DESCRIPTION
**Pull request by smitshah2531**
### What does this PR do?###

Updated the 'SFDX: Invoke Apex Test' to 'SFDX: Run Apex Test'

### What issues does this PR fix or reference?

@W-7561371@

### Functionality Before

Before_Invoke_Apex_Test

### Functionality After

Updated to Run_Apex_Test

### CODE REVIEW

File: packages/salesforcedx-vscode-core/package.nls.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "force_source_push_default_scratch_org_text": "SFDX: Push Source to Default Scratch Org",
   "force_source_push_force_default_scratch_org_text": "SFDX: Push Source to Default Scratch Org and Override Conflicts",
   "force_source_status_text": "SFDX: View All Changes (Local and in Default Scratch Org)",
-  "force_apex_test_run_text": "SFDX: Invoke Apex Tests...",
+  "force_apex_test_run_text": "SFDX: Run Apex Tests...",
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode-core/package.nls.json:13]: @hbc-bindu should the command text include the ellipses? or do we prefer it without? it doesn't look like the lightning run test command includes it

**Comment by smitshah2531** [comment]
> There are a few files under `docs/_articles` that need to be updated as well.

**Comment by smitshah2531** [comment]
Made a new commit after updating a doc. Please check over it and let me know if it needs any additional changes!!!

File: packages/salesforcedx-vscode-core/package.nls.json (hunk 1 above)
Comment (hbc-bindu) [reply packages/salesforcedx-vscode-core/package.nls.json:13]: Yes @AnanyaJha, we don't need ellipsis in the command text.

**Comment by smitshah2531** [comment]
Removed ellipsis from command text. Please check and let me know if need additional changes!

File: packages/salesforcedx-vscode-core/package.nls.json (hunk 1 above)
Comment (smitshah2531) [reply packages/salesforcedx-vscode-core/package.nls.json]: Changes Done. Please look over it.

File: docs/_articles/en/user-guide/vscode-commands.md [hunk 2]
```diff
@@ -135,7 +135,7 @@ Read on to learn about the commands available in Salesforce Extension for VS Cod
   - Fetches the most recent debug log from the default org.
   - Executes the CLI command `force:apex:log:get`
   - See [View Apex Debug Logs](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_develop_view_apex_debug_logs.htm).
-- **SFDX: Invoke Apex Tests...**
+- **SFDX: Run Apex Tests...**
```
Comment (AnanyaJha) [inline docs/_articles/en/user-guide/vscode-commands.md:138]: @smitshah2531 Let's remove the ellipsis here

File: docs/_articles/ja/user-guide/vscode-commands.md [hunk 3]
```diff
@@ -135,7 +135,7 @@ Read on to learn about the commands available in Salesforce Extension for VS Cod
   - Fetches the most recent debug log from the default org.
   - Executes the CLI command `force:apex:log:get`
   - See [View Apex Debug Logs](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_develop_view_apex_debug_logs.htm).
-- **SFDX: Invoke Apex Tests...**
+- **SFDX: Run Apex Tests...**
```
Comment (AnanyaJha) [inline docs/_articles/ja/user-guide/vscode-commands.md:138]: @smitshah2531 here as well!

File: packages/salesforcedx-vscode-core/package.nls.ja.json [hunk 4]
```diff
@@ -26,8 +26,8 @@
   "force_config_list_text": "SFDX: すべての設定変数を一覧表示",
   "force_alias_list_text": "SFDX: すべてのエイリアスを一覧表示",
   "force_org_display_default_text": "SFDX: デフォルトの組織の詳細を表示",
-  "force_org_display_username_text": "SFDX: 組織の詳細を表示...",
-  "force_data_soql_query_input_text": "SFDX: SOQL クエリを実行...",
+  "force_org_display_username_text": "SFDX: 組織の詳細を表示",
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode-core/package.nls.ja.json:29]: Let's not touch the other commands for now

File: docs/_articles/en/user-guide/vscode-commands.md (hunk 2 above)
Comment (smitshah2531) [reply docs/_articles/en/user-guide/vscode-commands.md]: Done!!!

File: docs/_articles/ja/user-guide/vscode-commands.md (hunk 3 above)
Comment (smitshah2531) [reply docs/_articles/ja/user-guide/vscode-commands.md]: Done!!!

File: packages/salesforcedx-vscode-core/package.nls.ja.json (hunk 4 above)
Comment (smitshah2531) [reply packages/salesforcedx-vscode-core/package.nls.ja.json]: Done!!!

**Pull request merged**